Dfe.Academies.External.Web/Models/ConversionApplication.cs

```diff
@@ -12,9 +13,19 @@ public class ConversionApplication
 
     public ApplicationTypes ApplicationType { get; set; }
 
-    public string? UserEmail { get; set; }
+    public ApplicationStatus ApplicationStatus { get; set; }
 
-    public string? Application { get; set; }
+	public string? UserEmail { get; set; }
+
+	/// <summary>
+	/// In format:- "Form a new single academy trust A2B_2"
+	/// </summary>
+	public string Application {
```

Comment:
Could we give this a different name? Maybe "ApplicationTitle"?

Reply:
good call, done